Makefile

```diff
@@ -25,7 +25,7 @@ help:
 
 	@echo "Environment Management --------------------------------------------------------"
 	@echo "  \033[1m\033[35mclean-venv\033[0m \033[37m(cv)\033[0m: \033[36mRemove virtual environment.\033[0m"
-	@echo "  \033[1m\033[35minstall\033[0m \033[37m(i)\033[0m: \033[36mInstall dependencies and tus_storagehandler.\033[0m"
+	@echo "  \033[1m\033[35minstall\033[0m \033[37m(i)\033[0m: \033[36mInstall dependencies and cloud_storage_handler.\033[0m"
```

Comment:
I guess this uses the project name from Cruft - which should be "Cloud Storage Handler"

Reply:
Btw, @JaeAeich: I think it would make much more sense to write:

> Install `{{ cookiecutter.project_title }}` and dependencies

rather than the other way around.

Or simply:

> Install package and dependencies

Not much added benefit of using a template var here, IMO.

Reply:
I'll try to remember this for the next Cookiecutter PR, but this is not a problem of this repo